projects/libs/core-library-typescript/src/virtual-machine/hard-coded-contracts/registry/event-counter-contract.ts

```diff
@@ -0,0 +1,25 @@
+import BaseSmartContract from "../base-smart-contact";
+
+export default class EventCounterContract extends BaseSmartContract {
+  public async reportEvent(event: string) {
+    const value: number = await this.getValue(event);
```

Comment:
you should verify that the 'event' is a string (in getCounter as well), see the kinatn (assuming thats the correct way of doing it)

Reply:
Done.